src/player/PlayerComponent.cpp

```diff
@@ -282,6 +297,13 @@ void PlayerComponent::queueMedia(const QString& url, const QVariantMap& options,
   extraArgs.insert("ad", "");
   extraArgs.insert("vd", "");
 
+  extraArgs.insert("tls-verify", "yes");
+  extraArgs.insert("stream-lavf-o", "verifyhost=" + host);
```

Comment:
These don't really need to be per-media. They can be set as global options. (The actual difference isn't huge, but still.)